lostmediafinder/finder.py

```diff
@@ -337,6 +337,32 @@ async def _run(cls, id, session: aiohttp.ClientSession):
             rawraw=raw, metaonly=False, comments=False, available=available
         )
 
+class removededm(YouTubeService):
+    """
+    Queries the removed.edm database for the video you requested.
+    """
+    name = methods["removededm"]["title"]
+    configId = "removededm"
+
+    @classmethod
+    async def _run(cls, id, session: aiohttp.ClientSession) -> T:
+        ismeta = False
+        lien = f"https://removededm.com/wiki/File:{id}.mp4" or f"https://removededm.com/wiki/File:{id}.webm"
+        async with session.head(lien, allow_redirects=False, timeout=15) as response:
+            redirect = response.headers.get("location")
+            archived = bool(redirect) or code == 200 # if there's a redirect, it's archived
```

Comment:
Are there any cases where the server redirects one of these to another page? I haven't been able to find any.